### ISSUE TITLE
Multiple pages for recommended videos and comments

### ISSUE THREAD

**Issue by Mcfly2340** [Frontesque/VueTube#138]
So when you're playing a video in portrait mode you can swipe to the right from the recommended videos section and see the comments in a list. and when you swipe to the left from the comments you are back to the recommended videos.

**Comment by Svk1190** [comment]
In this interface, it is not obvious where the comments are hidden. A new user will see the video recommendations and two dots, and he won't know where the comments are. After trying it out, he would learn it, but the initial friction is undesirable.

Suggestion: Add a button for comment. When user clicks it, it switches from the video recommendations to the comment page. Clicking on the same button again switches back to the video recommendations. The button label and logo also switches. The same button can be used for the switching action as long as it's text label is switched at the same time.

**Comment by Mcfly2340** [comment]
> Suggestion: Add a button for comment. When user clicks it, it switches from the video recommendations to the comment page. Clicking on the same button again switches back to the video recommendations. The button label and logo also switches. The same button can be used for the switching action as long as it's text label is switched at the same time.

This suggestion was why i opened this issue, because i heard some people complaining about the current UI of the yt app. In the current version of yt you have to click on the word comments to see the full list of comments, to make it easier i thought of a swipe to the right to open the comments. 
I get what you mean by the newer users having to figure it all out, but this suggestion could also be an option in the app itself for the users to configure. Just like instead of the dots, have it be an icon for recommended videos and comments.

**Comment by Svk1190** [comment]
Maybe we can ask the users about their preferences through UI vote.

**Comment by Mcfly2340** [comment]
> Maybe we can ask the users about their preferences through UI vote.

Yeah maybe, but I do think the user should get a choice if they want to stay with the default option or not

**Comment by Thestarchypotat** [comment]
What if you had a button that looked something like

_____________
|Comments> |
‐---------------------

And you can either click it or enable a setting like "swipe to comments" and then you can just swipe on the screen, and then it shows the comments and the button does a cool little animation (that you can turn on/off) like

___________
|com^reco|
-----------------

________________
|<reccomended|
--------------------------

Like the arrow slides over and flips at the same time if that makes sense?

**Comment by 404-Program-not-found** [comment]
I'll pick this up while I'm working on the comments itself. Perhaps it should be a setting?

**Comment by Mcfly2340** [comment]
> I'll pick this up while I'm working on the comments itself. Perhaps it should be a setting?

That would be amazing! My first thought was that it should be a choice, because there are still people who want them separated.
